### PR TITLE
NEWS.md: new file and updated for 2.59

### PR DESCRIPTION
**Pull request by mvo5**
This NEWS.md will replace the auto-generated changelogs and we shoudl update it on the fly for any significant PR we land (and it can contain more context than this initial very terse version).

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# New in snapd 2.59:
+
+* Extra kernel command line via snap configuration
+* Support for Full-Disk-Encryption using ICE
+* Support for arbitrary home dir locations via snap configuration
+* New nvidia-drivers-support interface
+* Support for udisks2 snap
+* Pre-download of snaps read for refresh and automatic refresh of the snap
```
Comment (MiguelPires) [inline NEWS.md:8]: Ready?
```suggestion
* Pre-download of snaps ready for refresh and automatic refresh of the snap
```

File: NEWS.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  when all apps are closed
+* New microovn interface
+* Support uboot with `CONFIG_SYS_REDUNDAND_ENV=n`
+* Make "snap-preseed --reset" re-exec when needed
+* Update the fwupd interface to support fully confined fwupd
+* The memory,cpu,thread quota options are no longer experimental
+* Support debugging snap client requests via the `SNAPD_CLIENT_DEBUG_HTTP`
+  environemnt variable
```
Comment (MiguelPires) [inline NEWS.md:16]:
```suggestion
  environment variable
```

File: NEWS.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# New in snapd 2.59:
+
+* Extra kernel command line via snap configuration
```
Comment (pedronis) [inline NEWS.md:3]: Support setting extra kernel command line parameters via snap configuration and under a gadget allow-list

**Review by pedronis — APPROVED** [review]
pushed one addition, thanks

**Pull request merged**